src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +70,81 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    
+}
+
+/**
+ * Represents the Block component of a Person's address in the address book.
+ */
+class Block{
```

Comment:
If a class is not a nested class, it is always good to have a separate file for the class. In this case nested class are ok but your implementation is not a nested implementation.